Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -570,44 +536,29 @@ public void PreProc_EndToEnd_Success_LocalSettingsAreUsedInSonarLintXML()
             AssertExpectedServerSetting("shared.CASING", "server upper case value", actualConfig);
         }
 
-        #endregion Tests
-
-        #region Setup
-
-        private IEnumerable<string> CreateArgs(string projectKey, string projectName, Dictionary<string, string> properties)
+        private static IEnumerable<string> CreateArgs(string projectKey = "key", string projectName = "name", string projectVersion = "1.0", string organization = null, Dictionary<string, string> properties = null)
```

Comment:
These parameters can be removed and hardcoded below. They are never set to non-default values
```suggestion
        private static IEnumerable<string> CreateArgs(string organization = null, Dictionary<string, string> properties = null)
```

Reply:
It will also enable to remove `organization:` fro the invocations, because it is obvious from the value